policy/modules/services/irqbalance.te

```diff
@@ -28,6 +28,7 @@ allow irqbalance_t self:capability { setpcap };
 dontaudit irqbalance_t self:capability sys_tty_config;
 allow irqbalance_t self:process { getcap getsched setcap signal_perms };
 allow irqbalance_t self:udp_socket create_socket_perms;
+allow irqbalance_t self:unix_stream_socket { accept listen };
```

Comment:
Should probably be:
```suggestion
allow irqbalance_t self:unix_stream_socket create_stream_socket_perms;
```
The other create_stream_socket_perms already come in from other interfaces, but this makes it explicit.